.github/workflows/ttrt-flatbuffer-ttnn.yml

```diff
@@ -0,0 +1,106 @@
+name: TTRT Flatbuffer TTNN Tests
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  schedule:
+    - cron: "0 */2 * * *" # This cron schedule runs the workflow every 2 hours
```

Comment:
Is this only for your testing?  Silicon tests should run on PR

Reply:
yes only for my testing